cmd/plasmacli/subcmd/eth/prove.go

```diff
@@ -76,28 +69,51 @@ var proveCmd = &cobra.Command{
 
 // Returns transaction results for given position
 // Trusts connected full node
-func getProof(ctx context.CLIContext, addr ethcmn.Address, position plasma.Position) (*tm.ResultTx, []byte, error) {
-	// query for the output
-	key := append(addr.Bytes(), position.Bytes()...)
-	res, err := ctx.QueryStore(key, "utxo")
+func getProof(position plasma.Position) (*tm.ResultTx, []byte, error) {
+	ctx := context.NewCLIContext().WithTrustNode(true)
+
+	key := store.GetOutputKey(position)
+	hash, err := ctx.QueryStore(key, "outputs")
 	if err != nil {
 		return &tm.ResultTx{}, nil, err
 	}
 
-	utxo := store.UTXO{}
-	if err := rlp.DecodeBytes(res, &utxo); err != nil {
-		return &tm.ResultTx{}, nil, err
+	txKey := store.GetTxKey(hash)
+	txBytes, err := ctx.QueryStore(txKey, "outputs")
```

Comment:
what do you think about having each store define it's name instead of hardcoding the string?